### PR TITLE
feat: `.get('*')` wildcard

### PR DESCRIPTION
**Pull request by AVVS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The version **must** be conform with the [semver](http://semver.org/) specificat
 
 #### `get(version)`
 Get a document from the store with the specified version.<br/>
-The version string could be a full version string or specify a range, such as `1.x`, in which case the highest version compatible will be returned.
+The version string could be a full version string or specify a range, such as `1.x`, in which case the highest version compatible will be returned. Specify `*` to get the highest version.
```
Comment (delvedor) [inline README.md]: Specify `*` to get the highest version *available*.

**Review by delvedor — COMMENTED** [review]
Great work! I love that you just needed to add one line to make it work :P

LGTM once you update the docs!

**Comment by AVVS** [comment]
done :)

**Review by delvedor — APPROVED** [review]
LGTM

**Pull request merged**